Core/DotLanches.Domain/Entities/Pedido.cs

```diff
@@ -1,6 +1,43 @@
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
+using DotLanches.Domain.Exceptions;
+
 namespace DotLanches.Domain.Entities;
 
 public class Pedido
 {
-    
+    public int Id { get; set; }
+    public DateTime CreatedAt { get; set; }
+    public string? ClienteCpf { get; set; }
+    public Status? Status { get; set; }
+    public decimal TotalPrice { get; set; }
+    public IEnumerable<Combo> Combos { get; set; }
+
+    private Pedido() { }
+
+    public Pedido(int id,
+                  DateTime createdAt,
+                  string? clienteCpf,
+                  IEnumerable<Combo> combos)
+    {
+        Id = id;
+        CreatedAt = createdAt;
+        ClienteCpf = clienteCpf;
+        Combos = combos;
+
```

Comment:
Os pedidos estão sendo criados sem um status e estão indo para o banco com essa prop vazia:
![image](https://github.com/lanchesjaamp/dotlanche/assets/32016451/b24c449d-ec95-4bfe-8f3c-59ce59eb0764)

Como um pedido novo sempre deveria começar num status inicial, poderíamos setar aqui no contrutor mesmo, inclusive sugiro utilizar Enum para o status ao invés de um ID, visto que não existe necessidade do status ser uma entidade:
```suggestion
Status = StatusPedido.PedidoConfirmado;
```